src/main/java/org/myrobotlab/inmoov/Vision.java

```diff
@@ -68,6 +86,11 @@ public OpenCVFilter setActiveFilter(String filterName) {
       instance.opencv.addFilter(filterName);
     }
     instance.opencv.setActiveFilter(filterName);
+    SwingGui gui = (SwingGui) Runtime.getService("gui");

```

Comment:
You should not be getting a handle to the gui here.  What's the point of this?  If you want to get the filter, that's defined by the opencv service, not the swing gui!

Reply:
The point is not to get a filter, but the current gui . To maximize the jframe after a filter Added. That will resolve a bug that overexpand the window after a filter is added